tests/UnifiedSpecTests/Util.php

```diff
@@ -134,8 +134,8 @@ final class Util
             'delete' => ['id'],
             'downloadByName' => ['filename', 'revision'],
             'download' => ['id'],
-            'uploadWithId' => ['id', 'filename', 'source', 'chunkSizeBytes', 'disableMD5', 'contentType', 'metadata'],
-            'upload' => ['filename', 'source', 'chunkSizeBytes', 'disableMD5', 'contentType', 'metadata'],
```

Comment:
Spec tests should be updated to remove this options.

Reply:
Thanks for the directions. The failing test is now skipped and I created [DRIVERS-2976](https://jira.mongodb.org/browse/DRIVERS-2976)